docker-compose.yml

```diff
@@ -41,5 +42,12 @@ services:
     ports:
       - 8081:8080
 
+  redis_app:
+    build:
+      context: .
+      dockerfile: Docker/Redis/Dockerfile
+    ports:
+      - "6379:6379"
```

Comment:
use the same style of declaring. without quotes

Reply:
done.